astrodb_utils/__init__.py

```diff
@@ -1,21 +1,24 @@
-import logging
+import logging  # noqa: I001

```

Comment:
Do you really need `noqa` here? `I001` sounds like a imports sorting error but this looks correct.